src/exporter/object.js

```diff
@@ -10,7 +9,7 @@ module.exports = (node, name, pathRest, ipldResolver, resolve) => {
   if (pathRest.length) {
     const pathElem = pathRest.shift()
     newNode = node[pathElem]
-    const newName = path.join(name, pathElem)
+    const newName = name + '/' + pathElem
```

Comment:
Seems slightly correct but not 100%. What if `name` ends with a slash or `pathElem` starts with a slash? In that case, we'll have double-slash, while path.join would handle it correctly (but not correctly on windows)

Reply:
Totally agree with you.  Ideally I should be using be `path.posix.join` which works just fine on Node.  However, in the Browser it fails with `TypeError: Cannot read property 'join' of undefined] `.  Must be a babel/webpack issue.

Reply:
Let's make sure to not let this one pass. Please add a test for @VictorBjelkholm's case and ensure it works both in browser and node.js

Reply:
The test case (guard) you want can never happen.  `name` is really a hash, so it can not have shash.  `pathElem` comes from `pathRest` which is an array that is path separated.